cesium_app/tests/fixtures.py

```diff
@@ -150,18 +149,20 @@ def create_test_model(fset, model_type='RandomForestClassifier'):
 
 
 @contextmanager
-def create_test_prediction(dataset, model):
+def create_test_prediction(dataset, featureset, model):
     """Create and yield test prediction, then delete.
 
     Params
     ------
     dataset : `models.Dataset` instance
-        The dataset on which prediction will be performed.
+        Dummy dataset used to create prediction instance.
+    featureset : `models.Featureset` instance
+        The featureset on which prediction will be performed.
     model : `models.Model` instance
         The model to use to create prediction.
 
     """
-    fset, data = featurize.load_featureset(model.featureset.file.uri)
+    fset, data = featurize.load_featureset(featureset.file.uri)
```

Comment:
See the new test - there are two feature sets created there - one labeled, used to build the model, the other unlabeled, used in the prediction.